addons/website_slides/models/slide_slide.py

```diff
@@ -667,6 +699,13 @@ def _action_set_quiz_done(self):
 
         return self.env.user.sudo().add_karma(points)
 
+    def action_view_embeds(self):
+        self.ensure_one()
+
+        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_embed_action")
+        action['context'] = {'search_default_slide_id': self.id}
```

Comment:
Maybe a ``create: False`` also ? Not sure we should create embed manually :) .
Edit: seems the view itself is no create, so ok then ^^